x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_anomaly_alert_type.ts

```diff
@@ -258,8 +255,8 @@ export function registerTransactionDurationAnomalyAlertType({
                 ...getEnvironmentEsField(environment),
                 [TRANSACTION_TYPE]: transactionType,
                 [PROCESSOR_EVENT]: ProcessorEvent.transaction,
-                [ALERT_SEVERITY_LEVEL]: severityLevel,
-                [ALERT_SEVERITY_VALUE]: score,
+                [ALERT_SEVERITY]: severityLevel,
+                // [ALERT_SEVERITY_VALUE]: score,
```

Comment:
I think we can remove this rather than commenting it out.

Reply:
Good catch!